api/lib/domain/usecases/start-writing-campaign-assessment-results-to-stream.js

```diff
@@ -125,18 +125,14 @@ function _createHeaderOfCSV(skills, competences, idPixLabel, organizationType, o
   ];
 }
 
-function _extractCompetences(allCompetences, skills) {
-  return _(skills)
-    .map('competenceId')
-    .uniq()
-    .map((competenceId) => {
-      const competence = _.find(allCompetences, { id: competenceId });
-      if (!competence) {
-        throw new Error(`Unknown competence ${competenceId}`);
-      }
-      return competence;
-    })
-    .value();
+function _extractTargetedCompetences(allCompetences, targetedCompetenceIds) {
+  return targetedCompetenceIds.map((competenceId) => {
+    const competence = _.find(allCompetences, { id: competenceId });
```

Comment:
```suggestion
    const competence = allCompetences.find(({ id} ) => id === competenceId);
```

Reply:
Tu trouves ça plus clair ?

Reply:
Je comprends qu'il y a eu une décision de se passer de lodash lorsque les méthodes natives permettent de le faire.

Reply:
Plus généralement, en ce qui me concerne, c'est pas vraiment une considération native / lodash. Je trouve ça plus clair qu'on ait l'air d'appliquer une fonction à la collection, plutôt que celle-ci soit passée en paramètre (if you know what i mean)